pyproject.toml

```diff
@@ -72,6 +72,7 @@ select = [
     "PERF",
     "PLE",
     "TRY302",
```

Comment:
Please keep in alphabetical order (above TRY)

Reply:
Sure. I also moved  UP assuming the whole block after the comment should be alphabetical.